neqo-transport/src/connection.rs

```diff
@@ -706,14 +682,18 @@ impl Connection {
         qinfo!("Sending VN Packet instead of normal output");
         let hdr = PacketHdr::new(
             0,
+            // Actual version we support and a greased value.
             PacketType::VN(vec![QUIC_VERSION, 0x4a4a_4a4a]),
             Some(0),
             recvd_hdr.scid.unwrap().clone(),
             Some(recvd_hdr.dcid.clone()),
             0, // unused
             0, // unused
         );
-        let cs = self.obtain_crypto_state(0).unwrap();
+
+        // Do not save any state when generating VN pkt, so cs is not
```

Comment:
The right fix here is to move the version negotiation stuff to the "Server" class.  But this is fine for the moment.

Reply:
yep.